android/app/build.gradle

```diff
@@ -25,8 +25,14 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
+def keystoreProperties = new Properties()
+def keystorePropertiesFile = rootProject.file('key.properties')
+if (keystorePropertiesFile.exists()) {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+}
+
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
```

Comment:
Can we go lower than this? I’m not sure about the current distribution of SDK versions but we want to provide the app to as many people as possible ofc.

Reply:
https://flutter.dev/docs/resources/faq#what-devices-and-os-versions-does-flutter-run-on

Reply:
That only determines which version of the SDK you have to use to build the application. `minSdkVersion 16` determines the minimum allowed version which matches the version supported by Flutter.